### ISSUE TITLE
Mac remote KDE, touch scrolling speed is too fast

### ISSUE THREAD

**Issue by little-snow-fox** [neutrinolabs/xrdp#1936]
Hi, I use Mac remote KDE, touch vertical scrolling is too fast, can it be modified by configuration?

System info:
```
snowfox@snowfox-work:~$ cat /etc/issue
KDE neon 5.22 \n \l

snowfox@snowfox-work:~$ cat /proc/version
Linux version 5.4.0-74-generic (buildd@lgw01-amd64-038) (gcc version 9.3.0 (Ubuntu 9.3.0-17ubuntu1~20.04)) #83-Ubuntu SMP Sat May 8 02:35:39 UTC 2021

snowfox@snowfox-work:~$ lsb_release -is
Neon
snowfox@snowfox-work:~$ lsb_release -rs
20.04

root@snowfox-work:/opt/sysbox# cat /etc/os-release 
NAME="KDE neon"
VERSION="5.22"
ID=neon
ID_LIKE="ubuntu debian"
PRETTY_NAME="KDE neon User Edition 5.22"
VARIANT="User Edition"
VARIANT_ID=user
VERSION_ID="20.04"
HOME_URL="https://neon.kde.org/"
SUPPORT_URL="https://neon.kde.org/"
BUG_REPORT_URL="https://bugs.kde.org/"
LOGO=start-here-kde-neon
PRIVACY_POLICY_URL="https://www.ubuntu.com/legal/terms-and-policies/privacy-policy"
VERSION_CODENAME=focal
UBUNTU_CODENAME=focal
```

Attempted to modify "Mouse wheel scrolls", nothing happened.  
<img width="1036" alt="截屏2021-06-30上午10 30 26" src="https://user-images.githubusercontent.com/23468639/123893068-49b34180-d98e-11eb-8e41-9eb19166d804.png">

Touchpad Configuration modifications are not allowed.
<img width="1035" alt="截屏2021-06-30上午10 30 39" src="https://user-images.githubusercontent.com/23468639/123893220-972fae80-d98e-11eb-9300-e51a87e9be71.png">

**Comment by seflerZ** [comment]
This is because xrdp converts the touch pad scrolling event as mouse wheel, but touch scrolling is continuous event, so xrdp send too many fake mouse wheel event to xorg. The solution is add a threshold to wheel down event or add a special handler for touch pad.

**Comment by metalefty** [comment]
@seflerZ has solved it in neutrinolabs/xorgxrdp#234, closing.
